cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,85 @@
+= VPC Upgrade to Private Space 
+
```

Comment:
Overall missing the quotas piece - we need to clarify that they are using doubled entitlements for the duration of their upgrade - for both VPC & Network Connections. Therefore, to ensure they stay compliant, they need to work with their account team on entitlements and finish their upgrade in a timely manner.